src/v2/Apps/Settings/Routes/EditSettings/SettingsEditSettingsInformation.tsx

```diff
@@ -25,149 +25,115 @@ export const SettingsEditSettingsInformation: React.FC<SettingsEditSettingsInfor
+            otherwise: field => field.notRequired(),
+          }),
+        })}
+        onSubmit={async ({ email, name, password, phone }) => {
+          try {
+            await submitUpdateSettingsInformation({
+              email,
+              name,
+              password,
+              phone,
+            })
+
+            sendToast({
+              variant: "success",
+              message: "Information updated successfully",
+            })
+          } catch (err) {
+            console.error(err)
+
+            sendToast({
+              variant: "error",
+              message: "There was a problem",
+              description: err.message,
+            })
+          }
         }}
-        validationSchema={ChangeUserInformationValidator}
-        onSubmit={onSubmit}
-        validateOnBlur={false}
       >
-        {({
-          errors,
-          handleBlur,
-          handleChange,
-          handleSubmit,
-          isSubmitting,
-          status,
-          touched,
-          values,
-        }) => (
-          <Form onSubmit={handleSubmit}>
+        {({ errors, handleBlur, handleChange, isSubmitting, values }) => (
+          <Form>
             <Join separator={<Spacer mt={2} />}>
               <Input
-                title="Full name"
+                title="Full Name"
                 name="name"
-                error={errors.name as any}
+                error={errors.name}
                 placeholder="Enter your full name"
                 value={values.name}
                 onChange={handleChange}
                 onBlur={handleBlur}
+                required
+                autoComplete="name"
               />
 
               <Input
                 title="Email"
                 name="email"
-                error={errors.email as any}
+                error={errors.email}
                 placeholder="Enter your email address"
                 value={values.email}
                 onChange={handleChange}
                 onBlur={handleBlur}
+                type="email"
+                required
+                autoComplete="email"
               />
 
               <Input
-                title="Mobile number"
+                title="Mobile Number"
                 name="phone"
                 placeholder="Enter your mobile phone number"
                 value={values.phone}
                 onChange={handleChange}
                 onBlur={handleBlur}
                 type="tel"
+                autoComplete="tel"
               />
 
               {me.paddleNumber && (
-                <>
-                  <Input
-                    name="paddleNumber"
-                    title="Bidder number"
-                    value={me.paddleNumber}
-                    readOnly
-                  />
-                </>
+                <Input
+                  name="paddleNumber"
+                  title="Bidder Number"
+                  value={"me.paddleNumber"}
```

Comment:
Actually wanted to ask about this section. In what instances is the `paddleNumber` present? No objections, just curious as to the purpose.

Reply:
Oh, woops your comment realized I also left this in a string (was just testing how it displayed).

You get a paddle number when you register for an auction.